Makefile

```diff
@@ -17,14 +17,17 @@ lint: format
 	poetry run flake8 aws_lambda_powertools/* tests/*
 
 test:
-	poetry run pytest -m "not perf" --cov=aws_lambda_powertools --cov-report=xml
+	poetry run pytest -m "not (perf or e2e)" --cov=aws_lambda_powertools --cov-report=xml
```

Comment:
In this case, ```-m "not (perf or e2e)"``` option doesn't work as expected. [Tests should be marked](https://docs.pytest.org/en/7.1.x/example/markers.html) with ```@pytest.mark.e2e```  for the ```-m``` option to work as expected. ```--ignore tests/e2e``` (leaving ```-m "not perf"``` intact) works just fine, or you need to mark all the e2e tests with the corresponding custom mark.

Reply:
will change to --ignore as it seems less error prone. Thx!